babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  presets: [
+    ['@babel/preset-env', {targets: {node: 'current'}}],
```

Comment:
Jest doesn't support TypeScript. Babel just converts .tsx files into .js for Jest